torch/linalg/__init__.py

```diff
@@ -140,3 +142,93 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+_add_docstr(_linalg.linalg_eigh, r"""
+linalg.eigh(input, UPLO='L') -> tuple(Tensor, Tensor)
```

Comment:
`UPLO -> upper`.

Reply:
`UPLO={'L', 'U'}` is used for for NumPy compatibility.